OpenRA.Mods.Common/Scripting/Global/BeaconGlobal.cs

```diff
@@ -25,6 +25,7 @@ public BeaconGlobal(ScriptContext context) : base(context)
 			radarPings = context.World.WorldActor.TraitOrDefault<RadarPings>();
 		}
 
+		[ScriptPropertyAISafe] // not 100% sure about safety but AI developers might make AI call for help
```

Comment:
Beacons are client side, so it won't desync but it also won't show for anybody except the game host.  Issue a `PlaceBeacon` order to the player actor if you want this to work everywhere.